src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -366,7 +365,7 @@ private void setWinningThreshold(Map<String, BigDecimal> currentRoundCandidateTo
                   ? config.getNumberOfWinners()
                   : config.getNumberOfWinners() + 1);
       // If we use integers, we shouldn't use any decimal places.
-      // Otherwise we use the amount of decimal places specified by the user
+      // Otherwise, we use the amount of decimal places specified by the user
```

Comment:
Nit: please add . at end while you're already here.

Reply:
Done.